setup.py

```diff
@@ -48,24 +48,26 @@ def run(self):
 
 
 setup(
-    name="darglint",
-    version="1.8.1",
+    name="darglint2",
+    version="2.0.0",
     author="Terrence Reilly",
     author_email="terrencepreilly@gmail.com",
+    maintainer="Antti Kaihola",
+    maintainer_email="13725+akaihola@users.noreply.github.com",
```

Comment:
@terrencepreilly, what's your preference for how to keep your name visible, and how and whether to keep your contact information there?

I'm also now started wondering about my e-mail address – I've been preferring to use the non-receiving GitHub e-mail address, but perhaps I really need to set up a dedicated e-mail address for my Open Source activities.

That said, I very much prefer to be contacted through GitHub instead of by e-mail.

Reply:
I'm not too particular about that.  You could just leave it in a comment, or the readme.  Or just remove it and link to the original repo on github.  It's probably best to not include my email, because I'll only check it once a week or so, and I don't want to give a false impression that I'd be very responsive.

Reply:
Ok thanks, I'll make sure to not include your e-mail, but I'll figure out a good way to keep your name prominently visible. I'll try to make what appears on the PyPI page match reality as well as possible.

Reply:
I'll do this in #17 and include it in the 1.8.3 release.